capture-server/src/server.rs

```diff
@@ -0,0 +1,50 @@
+use std::future::Future;
+use std::net::{SocketAddr, TcpListener};
+use std::sync::Arc;
+
+use time::Duration;
+
+use crate::config::Config;
+use capture::{billing_limits::BillingLimiter, redis::RedisClient, router, sink};
+
+pub async fn serve<F>(config: Config, listener: TcpListener, shutdown: F)
```

Comment:
If we're breaking this out of capture-server main, it might be better suited in capture, and then we don't need all of the internals to be public

Reply:
Will do!